hack/update-deps.sh

```diff
@@ -22,7 +22,7 @@ source $(git rev-parse --show-toplevel)/vendor/github.com/tektoncd/plumbing/scri
 
 cd ${REPO_ROOT_DIR}
 
-VERSION="release-1.8"
```

Comment:
can you add a comment or rename this variable to make it clear that it's the knative version?

Reply:
Done 👼🏼